### PR TITLE
feat(react-scheduler): add mobile adaptability to the Scheduler

### PR DESCRIPTION
**Pull request by AryamnovEugeniy**
Before:
image
image

After:
image
image

### CODE REVIEW

**Comment by MaximKudriavtsev** [comment]
Should we change the text size in the EditRecurrenceMenu?
image

**Comment by MaximKudriavtsev** [comment]
image

**Comment by MaximKudriavtsev** [comment]
image

File: packages/dx-react-scheduler-material-ui/src/templates/layouts/vertical-view-layout.jsx [hunk 1]
```diff
@@ -74,25 +80,24 @@ class VerticalViewLayoutBase extends React.PureComponent {
...
                 <DayScale />
-              </Grid>
+              </div>
             </Grid>
           </Grid>
 
           <Grid item xs="auto">
             <Grid container direction="row">
-              <Grid item xs={1}>
+              <div className={classes.timeScale}>
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler-material-ui/src/templates/layouts/vertical-view-layout.jsx]: Why `timeScale` applies to two places and one of them DayScale?

**Comment by AryamnovEugeniy** [comment]
FPR

File: packages/dx-react-scheduler-demos/src/demo-sources/scheduler-localization/material-ui/custom-formatting.jsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ import { appointments } from '../../../demo-data/appointments';
 const formatDayScaleDate = (date, options) => {
   const momentDate = moment(date);
   const { weekday } = options;
-  return momentDate.format(weekday ? 'dddd' : 'D');
+  return momentDate.format(weekday ? 'dd' : 'D');
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler-demos/src/demo-sources/scheduler-localization/material-ui/custom-formatting.jsx]: Maybe we just add a `text-overflow: elipsis` without change formating?

File: packages/dx-react-scheduler-material-ui/src/templates/edit-recurrence-menu/layout.jsx [hunk 3]
```diff
@@ -6,14 +6,33 @@ import DialogTitle from '@material-ui/core/DialogTitle';
 import Radio from '@material-ui/core/Radio';
 import RadioGroup from '@material-ui/core/RadioGroup';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
+import { withStyles } from '@material-ui/core/styles';
 
-export const Layout = React.memo(({
+const styles = ({ typography }) => ({
+  title: {
+    ...typography.h6,
+  },
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler-material-ui/src/templates/edit-recurrence-menu/layout.jsx]:
```js
title: typography.h6,
```

**Comment by AryamnovEugeniy** [comment]
FPR

**Pull request merged**